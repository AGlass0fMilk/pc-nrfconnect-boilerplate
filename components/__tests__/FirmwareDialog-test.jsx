import React from 'react';
import renderer from 'react-test-renderer';
import {
    decorateFirmwareDialog,
    mapFirmwareDialogDispatch,
    mapFirmwareDialogState,
} from '../FirmwareDialog';

describe('FirmwareDialog', () => {
    it('should implement component decorator', () => {
        expect(decorateFirmwareDialog).toBeInstanceOf(Function);
    });
    it('should implement dispatch mapper', () => {
        expect(mapFirmwareDialogDispatch).toBeInstanceOf(Function);
    });
    it('should implement state mapper', () => {
        expect(mapFirmwareDialogState).toBeInstanceOf(Function);
    });
});

describe('decorateFirmwareDialog', () => {
    it('should render component', () => {
        const FirmwareDialog = decorateFirmwareDialog('div');
        expect(renderer.create(
            <FirmwareDialog />,
        )).toMatchSnapshot();
    });
});
