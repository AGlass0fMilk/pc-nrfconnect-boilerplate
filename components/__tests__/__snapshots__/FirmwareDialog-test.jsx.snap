exports[`decorateFirmwareDialog should render component 1`] = `<div />`;
